Package.swift

```diff
@@ -92,6 +92,10 @@ let package = Package(
             name: "llbuildNinjaTests",
             dependencies: ["llbuildNinja", "gtest"],
             path: "unittests/Ninja"),
+        .testTarget(
+            name: "llbuildSwiftTests",
+            dependencies: ["llbuildSwift"],
+            path: "unittests/Swift"),
```

Comment:
Does it make more sense to put it outside of `unittests` to not confuse the gtest based tests?

Reply:
Unless it's causing an issue, I think it's fine as you have it.

Reply:
No, that seems fine

Reply:
I actually probably would like to move this, but let’s do it in a separate PR (I have other related things)

Reply:
I have another PR ready which builds on top of this by adding tests for the new Swift layer for the build database. Should we merge that as well before doing any work that refactors the testing infrastructure?